src/networks.ts

```diff
@@ -0,0 +1,119 @@
+type NetworkInfo = {
+  shortName: string;
+  chainID: number;
+  name: string;
+  currencySymbol: string;
+  baseAPI?: string;
+};
+
+export const networkInfo = new Map<number, NetworkInfo>([
+  [
+    1,
+    {
+      chainID: 1,
+      name: "Ethereum",
+      shortName: "eth",
+      currencySymbol: "ETH",
+      baseAPI: "https://safe-transaction.gnosis.io/api/v1",
+    },
+  ],
+  [
+    4,
+    {
+      chainID: 4,
+      name: "Rinkeby",
+      shortName: "rin",
+      currencySymbol: "RIN",
```

Comment:
I believe the native token on the test nets are all still ETH

Reply:
I was very confused by what standards are.

There is https://eips.ethereum.org/EIPS/eip-3770 this which is implemented by the gnosis safe.

There it says to checkout the shortnames the full list is defined in https://github.com/ethereum-lists/chains

There again it says
>There are also aggregated json files with all chains automatically assembled:
    - https://chainid.network/chains.json

In this json are definitions for each chainID. Including the currencySymbols. Thats where I found them. So I think this should be the correct currency. But I was confused at first too.

Reply:
It's fine. I call it rinkeby eth anyway so I guess it starts with RIN anyway. Must be metamask who is out of date.